src/test/java/com/mongodb/jdbc/MongoSQLConnectionTest.java

```diff
@@ -0,0 +1,159 @@
+package com.mongodb.jdbc;
```

Comment:
We need separate unit tests for MySQL and MongoSQL, unfortunately this code is very redundant